android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -1,9 +1,25 @@
 package io.logto.android
 
+import android.content.Context
 import io.logto.android.authflow.webview.WebViewAuthFlow
+import io.logto.android.model.Credential
+import io.logto.android.storage.CredentialStorage
 
 class Logto {
     companion object {
-        fun webViewAuthFlow() = WebViewAuthFlow()
+        lateinit var credentialStorage: CredentialStorage
+
+        fun webViewAuthFlow(context: Context): WebViewAuthFlow {
+            credentialStorage = CredentialStorage(context.applicationContext)
+            return WebViewAuthFlow(context, credentialStorage)
+        }
+
+        fun getCredential(): Credential? {
+            return if (this::credentialStorage.isInitialized) {
+                credentialStorage.getCredential()
+            } else {
+                null
+            }
```

Comment:
https://kotlinlang.org/docs/null-safety.html#safe-calls
感觉可以简化？

Reply:
This property is marked as lateinited, if we use safe-calls it would throw exception.
So it's not recommend to use before authenticated.
But after my consideration, we can't guarantee the usage.
make it optional is a better way :)

Reply:
good